templates/field/field--oe-featured-media.html.twig

```diff
@@ -0,0 +1,22 @@
+{#
```

Comment:
Might be better to provide a theme in the FeaturedMediaRenderedEntityFormatter that will invoke `@ecl-twig/media-container`? Otherwise we won't have advantages of using separate field formatter.

Reply:
ok